plugins/asus-hid/fu-asus-hid-plugin.c

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2024 Mario Limonciello <superm1@gmail.com>
+ * SPDX-License-Identifier: LGPL-2.1-or-later
+ */
+
+#include "config.h"
+
+#include "fu-asus-hid-device.h"
+#include "fu-asus-hid-firmware.h"
+#include "fu-asus-hid-plugin.h"
+
+struct _FuAsusHidPlugin {
+	FuPlugin parent_instance;
+};
+
+G_DEFINE_TYPE(FuAsusHidPlugin, fu_asus_hid_plugin, FU_TYPE_PLUGIN)
+
+static void
+fu_asus_hid_plugin_init(FuAsusHidPlugin *self)
+{
+}
+
+static void
+fu_asus_hid_plugin_constructed(GObject *obj)
+{
+	FuPlugin *plugin = FU_PLUGIN(obj);
+	FuContext *ctx = fu_plugin_get_context(plugin);
+	fu_plugin_add_device_gtype(plugin, FU_TYPE_ASUS_HID_DEVICE);
```

Comment:
and the child device for coverage?

Reply:
Ah; right.  I was originally going to have an E2E emulation test, but as we're merging sooner it's better to add the explicit coverage for those.